apps/finance/app/src/lib/verified-tokens.js

```diff
@@ -21,13 +21,16 @@ const MYCRYPTO_VERIFIED_TOKENS = [
   ['599GTO1', '0x5d9776472483ee2c2b204775547bff6db5a30fed'],
```

Comment:
@marczeller Could you also copy this file into the Agent app's list: https://github.com/aragon/aragon-apps/blob/master/apps/agent/app/src/lib/verified-tokens.js?

Reply:
of course, [PR done](https://github.com/aragon/aragon-apps/pull/1144). 

May I suggest Kleros Token TCR?, given both project history, they use Aragon for their governance, Aragon uses the TCR for curation, everybody wins.

but I guess this discussion is outside the immediate scope of this PR :joy:

Reply:
Haha yes, I was thinking that too, but do you have the Aave tokens listed on the Token TCR? Anyhow, that was definitely a resolver we'd like to build into https://github.com/aragon/use-token for verification purposes.

Reply:
CLA signed, yeah, I've added all Aave related tokens there. use-token() seems cool!

Reply:
If you want to query the tokens of the T2CR, it's pretty easy, you can look at this example: https://codesandbox.io/s/t2cr-and-badges-nyqi6?file=/src/index.js